core/mapper/exprmapper/expression/witype/type.go

```diff
@@ -0,0 +1,34 @@
+package witype
```

Comment:
why do we have a witype package? we should be using: core/data/types.go, no?

Reply:
This type more focus on expression and static functions, but yes, I will merge into data/types.go